src/DonationForms/resources/app/utilities/BillingAddressStateSelector.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+
+declare global {
```

Comment:
We can remove this file right?

Reply:
Resolved in d02f25bdff2aa059674415ad17c30bdad9657062